onnxruntime/core/graph/contrib_ops/contrib_defs.h

```diff
@@ -52,6 +52,7 @@ inline bool HasRawData(const ONNX_NAMESPACE::TensorProto& ten_proto) {
 void RegisterContribSchemas();
 void RegisterNchwcSchemas();
 void RegisterQuantizationSchemas();
+void RegisterQOrderedSchemas();
```

Comment:
Is this needed ? This is not defined anywhere.

Reply:
Nice finding!